sources/packages/api/src/route-controllers/education-program/education-program.controller.ts

```diff
@@ -125,13 +152,13 @@ export class EducationProgramController {
   }
 
   /**
-   * This retuns only the subset of the EducationProgram to get
+   * This returns only the subset of the EducationProgram to get
    * the complete EducationProgram DTO use the @Get(":id") method
    * @param programId
    * @returns
    */
   @AllowAuthorizedParty(AuthorizedParties.institution)
-  @Get(":programId/summary")
+  @Get(":programId/details")
```

Comment:
What was the reason behind the URL value change?

Reply:
yes, this controller actually returns a program details, not a program summary